assets/company-reports/actions.js

```diff
@@ -0,0 +1,82 @@
+import {errorHandler} from '../utils';
+import server from '../server';
+
+
+export const QUERY_REPORT = 'QUERY_REPORT';
+export function queryReport() {
+    return {type: QUERY_REPORT};
+}
+
+export const SET_ACTIVE_REPORT = 'SET_ACTIVE_REPORT';
+export function setActiveReport(data) {
```

Comment:
imo would be better to pass just report name and not the event object

Reply:
Yes, I have done that on `onChange` function